src/fields/countrySelector/field.ts

```diff
@@ -0,0 +1,18 @@
+import CountrySelectComponent from "./component";
+import { Field } from 'payload/types';
+
+
+const CountrySelector: Field = {
```

Comment:
```suggestion
const CountrySelectorField: Field = {
```
Я бы переназвал его, чтоб не путаться, что именно мы вызываем :)